openjdk/build.xml

```diff
@@ -24,6 +24,20 @@
 	<property name="DEST" value="${BUILD_ROOT}/openjdk" />
 	<property name="src" location="." />
 	<property environment="env" />
+	<if>
+		<or>
+			<equals arg1="${JDK_IMPL}" arg2="ibm"  />
+			<equals arg1="${JDK_IMPL}" arg2="openj9" />
+		</or>
+		<then>
+			<property name="skipDependency" value=false/>
+		</then>
+		<else>
+			<property name="skipDependency" value=true/>
+		</else>
+	</if>
+	<property name="LIB" value="tohandler_simple"/>
+	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml"/>
```

Comment:
I was thinking to have an ant target for this. And `dist` depends on `getDependentLibsCheck`

```
<target name="getDependentLibsCheck" >
  	<if>
		<or>
			<equals arg1="${JDK_IMPL}" arg2="ibm"  />
			<equals arg1="${JDK_IMPL}" arg2="openj9" />
		</or>
		<then>
			<property name="LIB" value="tohandler_simple"/>
	                <import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml"/>
                        <antcall target="getDependentLibs"/>
		</then>
	</if>
</target>
```

Reply:
updated :)